spec/koans.js

```diff
@@ -479,8 +494,8 @@ describe("the JavaScript language", function() {
       };
 
       var obj = new Obj();
-      //expect(obj.theName()).toEqual();
-      //expect(obj.theName).toBe(new Obj().theName);
+      expect(obj.theName()).toEqual('bob');
```

Comment:
This will actually give `undefined`, since name is defined as a local variable in the `Obj` function, and not as a property using `this.name = 'bob'`